apps/browser/src/platform/services/background-browser-biometrics.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from "@angular/core";
+
+import { NativeMessagingBackground } from "../../background/nativeMessaging.background";
+
+import { BrowserBiometricsService } from "./browser-biometrics.service";
+
+@Injectable()
+export class BackgroundBrowserBiometricsService extends BrowserBiometricsService {
+  constructor(private nativeMessagingBackground: NativeMessagingBackground) {
+    super();
+  }
```

Comment:
Just as info, another way to handle this is using `inject`.
```suggestion
    const nativeMessagingBackground = inject(NativeMessagingBackground);
```